databricks_cli/unity_catalog/catalog_cli.py

```diff
@@ -0,0 +1,145 @@
+# Databricks CLI
+# Copyright 2021 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+
+from databricks_cli.click_types import JsonClickType
+from databricks_cli.configure.config import provide_api_client, profile_option, debug_option
+from databricks_cli.unity_catalog.api import UnityCatalogApi
+from databricks_cli.unity_catalog.utils import mc_pretty_format
+from databricks_cli.utils import eat_exceptions, CONTEXT_SETTINGS, json_cli_base
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Create a new catalog.')
+@click.option('--name', required=True, help='Name of new catalog.')
+@click.option('--comment', default=None, required=False,
+              help='Free-form text description.')
+@click.option('--provider', default=None, required=False,
+              help='Name of the Provider (for creating Delta Sharing Catalog).')
+@click.option('--share', default=None, required=False,
+              help='Name of the Share under the Provider to create a Delta Sharing Catalog.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def create_catalog_cli(api_client, name, comment, provider, share):
+    """
+    Create a new catalog.
+
+    Calls the 'createCatalog' RPC endpoint of the Unity Catalog service.
```

Comment:
In general, the CLI docstring should mainly explain the side-effects and lack implementation details. Note that the first line of the docstring will be shown in the CLI.

Unless we need to explain some additional context, If the first-line of the docstring is already sufficient and the implementation is self-explanatory, it's okay to not have additional docstring after the first-line. You can refer to how other CLIs implement their docstring.

Reply:
I agree. I removed references to these RPCs and return values from all docstrings in #471.